src/database/DataTypes/Sensor.js

```diff
@@ -5,9 +5,199 @@
 
 import Realm from 'realm';
 
+import { NativeModules } from 'react-native';
+import { generateUUID } from 'react-native-database';
+
+const MANUFACTURER_ID = 307;
```

Comment:
I like what you've done with future proofing (incase we need to change sensor). This is just a thought (not going there for atm), you were probably thinking about this too -> manufacturer id into schema

Reply:
100%